dd-sdk-android/src/test/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewScopeTest.kt

```diff
@@ -1650,6 +1651,10 @@ internal class RumViewScopeTest {
         @BoolForgery waitForStop: Boolean,
         forge: Forge
     ) {
+
+        // this is a special case. if true, we need to deliver it anyway and test for it is below
+        assumeFalse(type == RumActionType.CUSTOM && !waitForStop)
```

Comment:
Instead you can remove the type forgery from the params and add 
```
val type = = forge.aValueFrom(RumActionType::class.java, exclude = listOf(RumActionType.CUSTOM))
```

Reply:
yes, I was thinking about it, but the thing is that combination of `CUSTOM + false` is valid for this test and I don't want to loose it.

Another solution will be to have dedicated tests for `CUSTOM` type: one for `true`, another for `false`.

Reply:
ended up with having 3 tests:

* non-custom `StartAction`
* Custom `StartAction` + `waitForStop=true`
* Custom `StartAction` + `waitForStop=false`